### ISSUE TITLE
Anemone doesn't theme any staged System App …

### ISSUE THREAD

**Issue by Aeneon** [AnemoneTeam/Anemone#280]
iOS 11 allows to remove some native Apps, like Contacts, FaceTime, Files and some more. After setting up iOS 11 the "stageable System Apps" were moved from "/private/var/staged_system_apps" to "/private/var/containers/Bundle/Application/" and a Subfolder with an GUID-String. With this new technique, Anemone doesn't apply any themed File for the following Apps :

- Music.app
- Videos.app
- Tips.app
- iBooks.app
- TV.app
- Reminders.app
- Podcasts.app
- MobileStore.app
- MobileNotes.app
- MobileMail.app
- MobileCal.app
- Maps.app
- Home.app
- FindMyFriends.app
- Files.app
- FaceTime.app
- Contacts.app
- Photo Booth.app
- News.app

If you want to theme only the Icon it works with IconBundles-Method, but not for any other File ( for Example the Play/Pause-Buttons in Music-App ). To blow up the Problem, Apple used the same BundleIdentifier for the App in "/private/var/containers/Bundle/Application/*" and also for the App-Placeholder, which can be found in "/System/Library/AppPlaceholders". The Placeholders were used after removing the corresponding App.

Hope this helps finding a Solution for this complex Problem …

**Comment by WhyNotCallMeRo** [comment]
I have loads of UI stuff themed in BuufJuiced and also MusicUI... It works but needs a ton of work....

**Comment by Aeneon** [comment]
Can you test this for me ? This Package contains some Images for Music-UI, which won't work on my iPad : [SpotiControls ( iOS 11.x ).theme.zip](https://github.com/AnemoneTeam/Anemone/files/1866516/SpotiControls.iOS.11.x.theme.zip) …

Which Bundle IDs do you use for BuufJuiced ?

**Comment by WhyNotCallMeRo** [comment]
Not at home.
Will reply later

Send with an iPhone X

**Comment by WhyNotCallMeRo** [comment]
Can’t test as my device only checks for @3x (iPhoneX).
I use same mediacontroles bundles but I have 167 files in Music UI…
Did you extract the ipsw and checked the asset.car files ??

Ro

**Comment by Aeneon** [comment]
Yeah extracted the IPSW and all Assets ( have a Tool written for this ). So can you send me your Music UI so I can test it on my Device ? I've downloaded BuufJuiced Base for iOS 10, but nothing was changed …

**Comment by WhyNotCallMeRo** [comment]
Old stuff..
I have 2018 version as free inside discord…
https://discord.gg/BD32Kt <https://discord.gg/BD32Kt>


Join and download the pinned Base-UI and/or Base-IB

I also have loads of adding but only for donators or vip….

**Comment by Aeneon** [comment]
Sorry, but I don't see any Change with iOS 11.0.3 on my iPad 5 …
![Screenshot](https://user-images.githubusercontent.com/15981999/38177462-47b8291e-3602-11e8-9ad3-22ea3e003c1a.jpg)

**Comment by WhyNotCallMeRo** [comment]
Hold on, I was still working on other bundles and hadn’t updated my latest progress.
The latest UI is uploading while I type…
Try again in 10 minutes….

Ro

**Comment by Aeneon** [comment]
Downloaded and installed it. Still the same Issue, CC is working, but nothing in the Music.app ( and also in the Files.app ) …

**Comment by WhyNotCallMeRo** [comment]
weird … all themed on my end…. Must be the diff. names @2x….

**Comment by Aeneon** [comment]
I don't think so. CC will be themed and I've tried with my Images ( and your Names ) to theme the UI. Looks like a Problem with the iPad or iOS 11.0.3. I don't know … very mysterious. But thanks for your Help …

**Comment by WhyNotCallMeRo** [comment]
Hold on. You are trying to theme the CC UI???

Send with an iPhone X

**Comment by Aeneon** [comment]
Yes, the CC works, but I want also to theme the Music-UI in Music-App itself …
